android/app/src/main/res/layout/group_message_list_item.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you feel so, go ahead. Just make sure that database is being updated correctly for different actions. Thank you! 🍒